vision/tests/system.py

```diff
@@ -720,6 +720,7 @@ def test_detect_web_images_from_content(self):
         web_images = image.detect_web(limit=limit)
         self._assert_web_images(web_images, limit)
 
+    @RetryErrors(unittest.TestCase.failureException)
```

Comment:
I can't say I care much for the approach:  it smells like papering over a set of bad assertions.